app/models/concerns/hud_reports/clients.rb

```diff
@@ -79,7 +79,7 @@ module HudReports::Clients
 
     private def annual_assessment_expected?(enrollment)
       return false if enrollment.last_date_in_program.present? &&
-        enrollment.last_date_in_program - enrollment.first_date_in_program < 1.year
+        enrollment.last_date_in_program - enrollment.first_date_in_program < 365
```

Comment:
On calculating anniversary dates, HUD says:

_It is important to use the “age” method of determining client anniversaries due to leap years; using “one year = 365 days” will eventually incorrectly offset the calculated anniversaries of long-term stayers._

Maybe
`return false if enrollment.first_date_in_program + 1.year > enrollment.last_date_in_program`?

Reply:
That's the ticket!

Reply:
Though in this case, we're just checking to see if it's been at least a year, so 365 will be wrong once every 4 years, but it won't get compounded.